src/styles/blocks/top-card.scss

```diff
@@ -0,0 +1,30 @@
+.card__img {
+  display: flex;
+  margin-bottom: 40px;
+  height: 134px;
+  width: 160px;
+}
+
+.card__description {
+  margin-bottom: 16px;
+  text-align: left;
+}
+
+.card__title {
+  color: $main-text-color;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  margin-bottom: 4px;
+}
+
+.card__product-code {
+  color: #616070;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+}
+
+.card:hover .card__title {
+  color: #34568b;
+}
```

Comment:
the elements should be placed in the same file as the block

Reply:
Not fixed